src/database/DataTypes/Period.js

```diff
@@ -5,19 +5,43 @@
 
 import Realm from 'realm';
 
+/**
+ * A period is a simple start and end date. Grouped by periodSchedules
+ * which are related to a MasterList and store. Used for applying
+ * settings from for a certain orderType.
+ *
+ * EG: A program - "HIV", a store "CHR" and the order type "Normal"
+ * may have a maxOrdersPerPeriod of 1.
+ * @property  {string} id
+ * @property  {string} name           A generic name field
+ * @property  {string} startDate      The starting date for the period
+ * @property  {string} endDate        The end date for this period
+ * @property  {string} periodSchedule The related schedule
+ * @property  {string} requisitions   related requisitions
+ */
 export class Period extends Realm.Object {
   numberOfRequisitions() {
     return this.requisitions.length;
   }
 
-  numberOfRequisitionsForProgram(program) {
-    return this.requisitions.filtered('program.id = $0', program.id).length;
+  requisitionsForOrderType(program, orderType) {
+    return this.requisitions.filtered(
+      'program.id = $0 && orderType = $1',
+      program.id,
+      orderType.name
+    ).length;
   }
 
   addRequisitionIfUnique(requisition) {
     if (this.requisitions.filtered('id == $0', requisition.id).length > 0) return;
     this.requisitions.push(requisition);
   }
+
+  toString() {
+    return `${this.startDate.toLocaleDateString('en-US')} - ${this.endDate.toLocaleDateString(
```

Comment:
On my machine it gives the same format. It will use the locale date and zone if no parameters are passed.
- Not sure if this should be consistent or something?
- Not sure if you're looking for a different format? (Oddly `toLocalDateString` is pretty customizable, but I can't find a way to change separators). Can make a method or use the other parsers we have?

Reply:
I think omitting parameter her would be ideal, it should default to system date:

Below, running `node` in terminal

<img width="286" alt="Screen Shot 2019-04-12 at 10 12 10 PM" src="https://user-images.githubusercontent.com/26194949/56030130-377a6800-5d70-11e9-91bc-ba9c97673936.png">

Reply:
done 2b9b707fc2635ca3a035716b75c97763ba36859b